### ISSUE TITLE
Buggy BelongsTo:  Edit page for a resource shows old field data.

### ISSUE THREAD

**Issue by johnmeehan** [thoughtbot/administrate#241]
Dropdown boxes retain/displaying old data.

The demo app does this aswell!
http://administrate-prototype.herokuapp.com/admin

*STEPS*
1.  Go to any dashboard and view an Order (any item with dropdown selects). 
2.  Press Edit and change any dropdown box (Customer).
3.  Press update.
4.  The show page show now has the new updated values. (DB values are also updated.)
5.  Click Edit again and the old values are back.

It just seems to happen on belongs to relationships.

**Comment by cjbell** [comment]
@johnmeehan a workaround is the following:

1) Run the generator for the views `rails generate administrate:views:field belongs_to` 
2) In the `_form.html.erb` file just generated, add the `id` as the second option for `options_for_select`

I did the following:

```
<%= f.select(field.permitted_attribute) do %>
   <%= options_for_select(field.associated_resource_options, field.data.try(:id)) %>
<% end %>
```

**Comment by graysonwright** [comment]
Fixed by #247.
